addons/mail/static/src/components/discuss/discuss_pinned_tests.js

```diff
@@ -0,0 +1,276 @@
+odoo.define('mail/static/src/components/discuss/discuss_pinned_tests.js', function (require) {
+'use strict';
+
+const {
+    afterEach: utilsAfterEach,
+    afterNextRender,
+    beforeEach: utilsBeforeEach,
+    start: utilsStart,
+} = require('mail/static/src/utils/test_utils.js');
+
+QUnit.module('mail', {}, function () {
+QUnit.module('components', {}, function () {
+QUnit.module('discuss', {}, function () {
+QUnit.module('discuss_pinned_tests.js', {
+    beforeEach() {
+        utilsBeforeEach(this);
+
+        this.start = async params => {
+            if (this.widget) {
+                this.widget.destroy();
+            }
```

Comment:
This is obsolete in start, it can be removed. You can remove it from wherever you copied it from too to avoid further mistake.